.travis.yml

```diff
@@ -1,5 +1,5 @@
 sudo: required
-dist: trusty
+dist: xenial
```

Comment:
this should not be needed as industrial_ci uses docker containers anyway

Reply:
Good to know. Will it use the corresponding Ubuntu LTS version? E.g. for melodic will it use Ubuntu bionic?